runtime/core/exec_aten/util/tensor_util.h

```diff
@@ -1120,6 +1120,29 @@ bool extract_scalar_tensor(executorch::aten::Tensor tensor, BOOL_T* out_val) {
   return true;
 }
 
+/**
+ * Maximum size of a string returned by tensor_shape_to_c_string, for
+ * stack allocation.
+ */
+constexpr size_t kTensorShapeStringSizeLimit = 1 + /* opening parenthesis */
+    10 * kTensorDimensionLimit + /* maximum digits we will print; update
```

Comment:
@dbort I am happy to lower digits per item if necessary. I don't know what amount of stack space I can reasonably expect to be available.

Reply:
What does this value work out to? If it's a couple hundred bytes it's probably ok

Reply:
kTensorDimensionLimit is 16, so it's 1 + 160 + 32 + 1 = 194 bytes

Reply:
That seems ok, especially if callers guard the definition of those stack buffers with an ifdef that will compile them out when logging is disabled